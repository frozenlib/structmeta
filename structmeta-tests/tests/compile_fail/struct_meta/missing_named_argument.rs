#[structmeta_tests::attr_required_name_value()]
fn my_func() {}

fn main() {}
