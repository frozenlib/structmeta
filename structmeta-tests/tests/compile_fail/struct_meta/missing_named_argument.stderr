error: missing argument `value`
 --> $DIR/missing_named_argument.rs:1:1
  |
1 | #[structmeta_tests::attr_required_name_value()]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
